config/routes.js

```diff
@@ -56,7 +56,7 @@ module.exports.routes = {
   'post /api/v1/pokemon/multi': 'PokemonController.uploadMultipleFiles',
 
   'get /api/v1/pokemon/:id': 'PokemonController.get',
-  'get /api/v1/pokemon/:id/pk6': 'PokemonController.download',
+  'get /api/v1/pokemon/:id/pk': 'PokemonController.download',
```

Comment:
Raw doesn't tell you what you are getting out. And pk is more consistent with what we had before. I will stick with this.